common/lib/dependabot.rb

```diff
@@ -2,5 +2,5 @@
 # frozen_string_literal: true
 
 module Dependabot
-  VERSION = "0.260.0"
+  VERSION = "0.259.0"
```

Comment:
This downgrades the version, seems like this is on the wrong end of a merge conflict

Reply:
fixed.  Goodd catch.

Reply:
It does not seem to be fixed